Sources/Plasma/NucleusLib/pnAsyncCoreExe/Pch.h

```diff
@@ -56,28 +56,21 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "plProduct.h"
 #include "hsThread.h"
-#include "hsWindows.h"
 
 #include "pnUtils/pnUtils.h"
 #include "pnNetBase/pnNetBase.h"
 #include "pnAsyncCore/pnAsyncCore.h"
 
 #include "Private/pnAceInt.h"
-#include "Private/Nt/pnAceNt.h"
 
+// Target Windows 7 or later for windows socket compatibility
+// ASIO needs to know this to warn about some API incompatibilities
+#define _WIN32_WINNT 0x0601
```

Comment:
Redefining `_WIN32_WINNT` outside of hsWindows.h, eh? I see you like living dangerously.-- This macro is defined as 0x501 in hsWindows.h. I would just bump it there and move along with life.

Reply:
Is there any actual need to keep it consistent?  Also, `hsWindows.h` does NOT set `_WIN32_WINNT` except on MinGW; otherwise it gets the default `windows.h` value of `0x0A00` (Windows 10) with the current VC2017 and 2019 SDKs